src/Pages/People/People.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Person } from '../../types';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isSuccess, setIsSuccess] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const fetchPeople = async () => {
+    try {
+      setIsLoading(true);
+      const newPeople = await getPeople();
+
+      setPeople(newPeople);
+    } catch {
+      setIsSuccess(false);
```

Comment:
will be better to name the variable related to the error, instead of `success`
for now it's not clear why didn't you set `true` for this variable after successful response